pkg/quantile/agent.go

```diff
@@ -50,14 +54,24 @@ func (a *Agent) Reset() {
 }
 
 // Insert v into the sketch.
-func (a *Agent) Insert(v float64) {
-	a.Sketch.Basic.Insert(v)
+func (a *Agent) Insert(v float64, sampleRate float64) {
```

Comment:
I think it's now a question of whether we want to pull this up into the sketchMap or leave it hidden here @kenny-house

Reply:
Since sample rate is part of the API for other metric types, making this part of the API for the `quantile.Agent` makes sense to me.